R/servers.R

```diff
@@ -185,18 +218,43 @@ serverInfo <- function(name) {
   info
 }
 
+#' @rdname servers
+#' @export
+addServerCertificate <- function(name, certificate, quiet = FALSE) {
+  # read the existing server information (throws an error on failure)
+  info <- serverInfo(name)
+
+  # append the certificate and re-write the server information
+  info$certificate <- inferCertificateContents(certificate)
+  write.dcf(info, serverConfigFile(name))
```

Comment:
Do we need to worry about the whitespace wrapping that `write.dcf()` does here? (just want to make sure that this won't potentially mangle long certificates; presumedly this isn't a problem as long as they don't contain any whitespace)

Reply:
Yes, we should probably use `keep.white` just in case (I'll add that).